Sources/Turf/Turf.swift

```diff
@@ -41,4 +41,25 @@ public struct Turf {
         let intersectsWithLine2 = b > 0 && b < 1
         return intersectsWithLine1 && intersectsWithLine2 ? intersection : nil
     }
+    
+    /**
+     Returns a new LineString based on bezier transformation of the input line
+     */
+    public static func bezier(_ line: LineString, resolution: Int = 10000, sharpness: Double = 0.85) -> LineString? {
```

Comment:
This static function would probably be a better fit as a method `LineString.bezier(resolution:sharpness)` in `LineString.swift`.

Reply:
Done as LineString extension in Spline.swift